presto-main/src/main/java/io/prestosql/server/ServerInfoResource.java

```diff
@@ -46,25 +47,24 @@
     private final NodeVersion version;
     private final String environment;
     private final boolean coordinator;
-    private final StaticCatalogStore catalogStore;
     private final GracefulShutdownHandler shutdownHandler;
     private final long startTime = System.nanoTime();
+    private final AtomicBoolean startupComplete = new AtomicBoolean(false);
```

Comment:
Our coding convention is to not set the initial value for atomics to the default, the same as with primitives. So remove the `false` since that is the default.